homeassistant/components/yamaha_musiccast/media_player.py

```diff
@@ -365,6 +376,15 @@ async def async_play_media(self, media_type: str, media_id: str, **kwargs) -> No
 
     async def async_browse_media(self, media_content_type=None, media_content_id=None):
         """Implement the websocket media browsing helper."""
+        if media_source.is_media_source_id(media_content_id):
+            return await media_source.async_browse_media(
+                self.hass,
+                media_content_id,
+                content_filter=lambda item: item.media_content_type.startswith(
+                    "audio/"
+                ),
+            )
```

Comment:
Fixed

Reply:
And works as expected. 
At the moment we only support browse media, if the device is turned on, because the device does not return any list data, if it is turned off. Should we change this behavior and instead turn the device on automatically when it's necessary? Otherwise only devices, which are already turned on, will be shown in the list of available players in the media browser.

Reply:
the SUPPORT_BROWSE feature flag shouldn't change while the entity is configured. I think that turning it on is indeed the right approach. However, I won't add this to this PR as I can't test it. Better if you open it 👍 you can tag me for review.